.github/workflows/integration-tests-reusable.yml

```diff
@@ -120,8 +132,8 @@ jobs:
       env:
         XUNIT_SHARD: ${{ matrix.shard }}
         XUNIT_SHARD_COUNT: ${{ inputs.num_shards }}
-        DAFNY_RELEASE: ${{ github.workspace }}/unzippedRelease/dafny
       run: |
+        echo ${{ inputs.all_platforms }} | grep true && export DAFNY_RELEASE="${{ github.workspace }}/unzippedRelease/dafny"
```

Comment:
This might be somewhat cleaner as a regular string comparison, but this is probably fine?